applications/luci-app-libreswan/htdocs/luci-static/resources/view/libreswan/proposals.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+'require view';
+'require ui';
+'require form';
+
+return view.extend({
+	render: function() {
+		var m, s, o;
+
+		m = new form.Map('libreswan', _('IPSec Proposals'));
+
+		s = m.section(form.GridSection, 'crypto_proposal');
+		s.anonymous = false;
+		s.addremove = true;
+		s.nodescriptions = false;
+		s.addbtntitle = _('Add Proposal');
+
+		o = s.tab('general', _('General'));
+
+		o = s.taboption('general', form.MultiValue, 'hash_algorithm', _('Hash Algorithm'));
+		o.default = 'md5';
+		o.value('md5', _('MD5'));
+		o.value('sha1', _('SHA1'));
+		o.value('sha256', _('SHA256'));
+		o.value('sha384', _('SHA384'));
+		o.value('sha512', _('SHA512'));
+
+		o = s.taboption('general', form.MultiValue, 'encryption_algorithm', _('Encryption Method'));
+		o.default = '3des';
```

Comment:
Please change this default, and mark 3des with a `*`, and a GUI note explaining it as unsafe.

Reply:
Updated